pyproject.toml

```diff
@@ -22,6 +22,11 @@ license = "MIT"
 readme = "README.md"
 packages = [{ include = "graphrag" }]
 
+[[tool.poetry.source]]
+name = "tsinghua"
+url = "https://pypi.tuna.tsinghua.edu.cn/simple"
```

Comment:
I understand that this may be needed for your use case but we can’t accept this change into main as it would redirect all users to the wrong location